extensions/jackson/deployment/src/main/java/org/apache/camel/quarkus/component/jackson/JacksonProcessor.java

```diff
@@ -37,7 +36,9 @@ FeatureBuildItem feature() {
     List<ReflectiveClassBuildItem> registerReflectiveClasses() {
         List<ReflectiveClassBuildItem> items = new ArrayList<>();
         items.add(new ReflectiveClassBuildItem(false, true, "com.fasterxml.jackson.module.jaxb.JaxbAnnotationModule"));
-        items.add(new ReflectiveClassBuildItem(false, false, JsonNode.class));
+        items.add(new ReflectiveClassBuildItem(false, false, com.fasterxml.jackson.databind.JsonNode.class));
```

Comment:
It could use class name here.
```suggestion
        items.add(new ReflectiveClassBuildItem(false, false, "com.fasterxml.jackson.databind.JsonNode"));
```

Reply:
Out of curiosity, the advantage to use the class name as string, is to not have the hard dependency ?

Reply:
In the first place, strings allow registering also non-public classes that might be inaccessible from here. Second, a string class name allows to avoid loading the class at build time which might sometimes come in handy, e.g. if the given class initializer does something nasty. Not loading the class may make the build time processing generally faster, but I never collected any numbers. Referencing the classes is of course better for maintenance, because if the class is renamed or removed, the compilation fails.